src/woo_publications/publications/tests/test_admin_logging.py

```diff
@@ -221,7 +221,7 @@ def test_admin_inline_update_admin(self):
         self.assertEqual(response.status_code, 302)
 
         with self.subTest("update inline update logging"):
-            log = TimelineLogProxy.objects.filter(object_id=document.pk).first()
+            log = TimelineLogProxy.objects.filter(object_id=str(document.pk)).first()
             assert log is not None
```

Comment:
I think this suffers from the missing `content_type` filter kwarg too?

And can this also be replaced with a `.get()` instead?